.gitignore

```diff
@@ -17,4 +17,5 @@ rm.**
 venv/
 terraform.tfstate
 .venv
-
+client/src/.featureform
```

Comment:
You should make this **/.featureform instead so it isnt neccesarily in that path

Reply:
Also make sure that .featureform directory is removed. I think the directory still exists on this branch

Reply:
Will change it. I dont see the directory anymore?